modules/core/src/wallet.ts

```diff
@@ -1363,19 +1364,16 @@ Wallet.prototype.accelerateTransaction = function accelerateTransaction(params,
    * @param parentTxId The ID of the transaction to get the full hex of
    * @returns {Bluebird<any>} The full hex for the specified transaction
    */
```

Comment:
Comment above still says "smartbit.com.au"

Reply:
Couldn't we just get it from the v2 indexer instead?

Reply:
Yeah, we should do this, great idea.

Reply:
This turned out to be a bit more difficult than I thought, so I deferred this for later work.